tests/python/contrib/test_ethosn/test_partition_params.py

```diff
@@ -0,0 +1,104 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Ethos-N partition parameter tests"""
+
+import pytest
+import tvm
+from tvm import relay
+import numpy as np
+
+from tvm.relay.op.contrib.ethosn import partition_for_ethosn77
+from tvm.relay.op.contrib.ethosn import partition_for_ethosn78
+
+fp32 = "float32"
+
+
+def test_ethosn78_partition_no_error():
+    a = relay.var("a", shape=[2, 7, 8, 8], dtype=fp32)
+    w = relay.const(np.random.uniform(-10, 10, (8, 7, 3, 3)).astype(fp32))
+    res = relay.nn.conv2d(a, w, kernel_size=(3, 3), padding=(1, 1), channels=8, out_dtype=fp32)
+    b = relay.var("b", shape=[8], dtype=fp32)
+    res = relay.nn.bias_add(res, b, axis=1)
+
+    mod = tvm.IRModule.from_expr(res)
```

Comment:
This looks like the same module repeated in the various tests, it'd be worth wrapping this in a `pytest.fixture`, such as:

```
@pytest.fixture
def simple_mod():
    a = relay.var("a", shape=[2, 7, 8, 8], dtype=fp32)
    w = relay.const(np.random.uniform(-10, 10, (8, 7, 3, 3)).astype(fp32))
    res = relay.nn.conv2d(a, w, kernel_size=(3, 3), padding=(1, 1), channels=8, out_dtype=fp32)
    b = relay.var("b", shape=[8], dtype=fp32)
    res = relay.nn.bias_add(res, b, axis=1)

    return tvm.IRModule.from_expr(res)

 def test_ethosn78_partition_no_error(simple_mod):
    opts = {"variant": "Ethos-N78"}
    partition_for_ethosn78(simple_mod, **opts)
```

Reply:
Done